src/esp/bindings_js/utils/utils.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
There's a lot of duplication here with `bindings_js/modules/utils.js`. It's confusing to have both of these in the library codebase. By "library codebase", I mean anything under habitat-sim/src, which we'll expect all webapps to share.

Can you find a way to consolidate down to just one `utils.js`? If it's not possible, it'll be better to include the version you need inside your web app (under `examples`). We shouldn't have two copies inside the library codebase.